packages/openactive-integration-tests/test/helpers/flow-stages/fetch-opportunities.js

```diff
@@ -99,16 +130,36 @@ async function runFetchOpportunities({ orderItemCriteriaList, requestHelper }) {
   //
   // Now that we have references to some opportunities that have been found or
   // created and match our criteria, let's get the full opportunities
-  const opportunityFeedUpdateResult = await OpportunityFeedUpdateFlowStage.run({
+  // const opportunityFeedUpdateResult = await OpportunityFeedUpdateFlowStage.run({
+  const opportunityFeedExtractResponses = await fetchOpportunityFeedExtractResponses({
     testInterfaceOpportunities,
-    orderItemCriteriaList,
     requestHelper,
+    useCacheIfAvailable: true,
+  });
+
+  // ## Create OrderItem for each Opportunity
+  const orderItems = opportunityFeedExtractResponses.map((opportunityFeedExtractResponse, i) => {
+    if (opportunityFeedExtractResponse && isResponse20x(opportunityFeedExtractResponse)) {
+      const acceptedOffer = getRandomRelevantOffer(opportunityFeedExtractResponse.body.data, orderItemCriteriaList[i].opportunityCriteria);
+      if (acceptedOffer === null) {
+        throw new Error(`Opportunity for OrderItem ${i} did not have a relevant offer for the specified testOpportunityCriteria: ${orderItemCriteriaList[i].opportunityCriteria}`);
+      }
+      return {
+        position: i,
+        orderedItem: opportunityFeedExtractResponse.body.data,
+        acceptedOffer,
+        'test:primary': orderItemCriteriaList[i].primary,
+        'test:control': orderItemCriteriaList[i].control,
+      };
+    }
+    return null;
```

Comment:
This was previously in opportunity-feed-update. I moved it here because it's only relevant to the initial opportunity fetch